puppetdb/templates/puppetdb-configmap.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service }}
 data:
   puppetdb.ini: |-
-{{ .Values.puppetdb_ini | indent 4 }}
+{{ .Values.puppetdb.puppetdb_ini | indent 4 }}
```

Comment:
This is a breaking change for the current deployment I think.

Reply:
I think not